docs/content/documentation/tooldevelopers/graphdatastructure/layoutoptions.md

```diff
@@ -90,6 +90,17 @@ This means that you can define multiple `IProperty` instances that describe the
 That is the reason why the first example above does not use `CoreOptions.DEBUG_MODE` to retrieve the property value, but `MyAlgorithmOptions.DEBUG_MODE` (where `MyAlgorithmOptions` is the metadata class generated for your layout algorithm).
 
 
+### Checking for the Presence of Property Values
+
+If we can always call `getProperty(...)` on a graph element, we may not be able to distinguish a default value from a value explicitly set on that element. If we need to, we can check whether a property was explicitly set through `hasProperty(...)`:
```

Comment:
The first _can_ feels a bit awkward to me. Maybe rephrase?

Reply:
Done, thanks. :)